### DISCUSSION TITLE
Bad IMS/INM caching results in group test report

### DISCUSSION THREAD

**Discussion by gingerbeardman** [FreshRSS/FreshRSS#6693] [Q&A] [answered]
> FreshRSS/1.24.0. This one latched bad IMS/INM values back in *May* and is still sending them. Hella broken.

> FreshRSS/1.24.0, 1.24.1. Bad caching, bad timing.

> FreshRSS/1.23.1. Bad caching, like the others.

- IMS = If-Modified-Since header
- INM = If-None-Match header

Links
- https://rachelbythebay.com/w/2024/08/02/fs/
- https://rachelbythebay.com/fs/help.html

I think you can contact the author for results data if you wish to see more or act on it

Here's the result page for NetNewsWire
- https://rachelbythebay.com/api/fsr/4467c09c31370b8375c6b2a320aa77cd5d999cd4

**Comment by Alkarex** [comment, marked as answer]
Fixed in FreshRSS 1.24.2+
https://github.com/FreshRSS/FreshRSS/issues/6712
https://github.com/FreshRSS/FreshRSS/pull/6723

**Reply by gingerbeardman** [reply]
Super, thanks!

I also thought the original blog post was un-actionable, hence me starting a discussion rather than an issue.
